doc/diagnostics/AddingTracepoints.md

```diff
@@ -67,7 +67,20 @@ For example:
 
 This breaks down as follows:
 
-`TraceEvent=Trc_VM_getMethodOrFieldID_dereferencedClass`  This specifies the type and the name of the tracepoint. `Trc_VM_getMethodOrFieldID_dereferencedClass` is the name of the generated macro to use in your code. The `TraceEvent=` denotes this is an event trace point. Other tracepoint types are:  `TraceEntry` `TraceExit` `TraceException` `TraceExit-Exception` `TraceAssert` `TraceDebug`.
+`TraceEvent=Trc_VM_getMethodOrFieldID_dereferencedClass`  This specifies the type and the name of the tracepoint. `Trc_VM_getMethodOrFieldID_dereferencedClass` is the name of the generated macro to use in your code. The `TraceEvent=` denotes this is an event trace point. Other tracepoint types are:
+
+- `TraceEntry`, `TraceEntry-Exception`:
+  Tracepoint to be placed at the entry point of a non-trivial function.
+- `TraceExit`, `TraceExit-Exception`:
+  Tracepoint for the exit points of a non-trivial function that has a `TraceEntry` tracepoint. Entry and Exit tracepoints facilitate indented formatting. If Entry and Exit tracepoints are unbalanced, the formatted trace may be indented incorrectly.
+- `TraceEvent`:
+  A generic tracepoint
+- `TraceException`:
+  Tracepoint is for out-of-line/error situations (not just Java exceptions)
+- `TraceAssert`:
+  An assertion tracepoint
```

Comment:
These points should end with a period (like the others).

Reply:
Added the period